tests/coalaJSONTest.py

```diff
@@ -56,7 +57,9 @@ def test_fail_acquire_settings(self):
 
     def test_show_all_bears(self):
         with bear_test_module():
-            retval, output = execute_coala(coala.main, 'coala', '--json', '-B')
+            retval, output = execute_coala(coala.main,
+                                           'coala', '--json', '-B',
+                                           stdout_only=True)
```

Comment:
When I remove this, the stderr stream contains:
`Implicit 'Default' section inheritance is deprecated. It will be removed soon. To silence this warning remove settings in the 'Default' section from your coafile. You can use dots to specify inheritance: the section 'all.python' will inherit all settings from 'all'.`

This is caused by the repos `.coafile`, which still has the deprecated `[Default]` section.
I think a better solution is to add command line argument `-I`, so that this and future changes to `.coafile` dont break this test case.
This is especially true because here we are using `bear_test_module`, which provides a different set of bears than are used by the repo `.coafile` , so there really should be lots of errors if the repo `.coafile` is not ignored.